linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_ownership/bash/shared.sh

```diff
@@ -4,29 +4,23 @@
 # complexity = high
 # disruption = medium
 
-# Declare array to hold list of RPM packages we need to correct permissions for
-SETPERMS_RPM_LIST=()
+# Declare array to hold set of RPM packages we need to correct permissions for
+declare -A SETPERMS_RPM_DICT
```

Comment:
Changes here are very similar to changes that were done to the tests/data/group_system/group_software/group_integrity/group_software-integrity/group_rpm_verification/rule_rpm_verify_permissions/all_permissions_ok.pass.sh test scenario earier.